Makefile.PL

```diff
@@ -30,6 +30,10 @@ WriteMakefile(
                 web  => 'https://github.com/jettero/term--ansicolorx',
             }
         },
+        x_authority    => "cpan:JETTERO",
```

Comment:
I might want to tweak this line to an actual email... I can't seem to find any docs on this meta format though. Where is this documented?

Reply:
Sorry for the delayed answer.
Well, actually I'm not able to find the origins of this field, but it's mentioned at least inside unofficial pause rules: https://gist.github.com/dagolden/9531860 written by David Golden, the author of CPAN::Meta::Spec.